### PR TITLE
ubah navbar

### PR DESCRIPTION
**Pull request by randy-ar**
Ki coba maneh komen naon weh tentang codingan urang nu diubah.

### CODE REVIEW

**Comment by Leblxnc** [comment]
Ambasing

File: src/layouts/Navbar.jsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ class NavbarLayout extends React.Component {
   render(){
     return (
       <nav className="navbar">
-        <h1>Aplikasi Catatan</h1>
+        <h1>Aplikasi Catatan Menggunakan React</h1>
```
Comment (Leblxnc) [inline src/layouts/Navbar.jsx:10]: ambasing

**Review by Leblxnc — CHANGES_REQUESTED** [review]
ambatukam

File: src/layouts/Navbar.jsx (hunk 1 above)
Comment (randy-ar) [reply src/layouts/Navbar.jsx:10]: sip aya ki

Comment (randy-ar) [reply src/layouts/Navbar.jsx:10]: urng coba ganti navbar na jadi ambasing

File: src/layouts/Navbar.jsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ class NavbarLayout extends React.Component {
   render(){
     return (
       <nav className="navbar">
-        <h1>Aplikasi Catatan Menggunakan React</h1>
+        <h1>Ambasing</h1>
```
Comment (Leblxnc) [inline src/layouts/Navbar.jsx:10]: sip

**Review by Leblxnc — APPROVED** [review]

**Comment by randy-ar** [comment]
sip urng merge kalau begitu

**Pull request merged**